projectile.el

```diff
@@ -1480,6 +1481,7 @@ With a prefix ARG invalidates the cache first."
 (defvar projectile-haskell-cabal '("*.cabal"))
 (defvar projectile-rust-cargo '("Cargo.toml"))
 (defvar projectile-r '("DESCRIPTION"))
+(defvar projectile-npm '("package.json" "node_modules"))
```

Comment:
Yeah it's hard and annoying that they are all using `package.json`, however the node_modules directory is only available after the first run of npm install which means that if you clone a repo using magit, open it with projectile and expect to be able to "compile" with C-c p c you'll not get what you expect.